android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -49,6 +51,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
 
+        downloadCompleteHandler = new DownloadCompleteHandler(getContext(), R.string.menu_feed);
```

Comment:
The DownloadCompleteHandler is made to show dialog of completion for the Event API, not sure why being used in Feed?